src/test/scala/edu/ie3/simona/model/participant2/HpModelSpec.scala

```diff
@@ -526,10 +764,102 @@ class HpModelSpec
               minPower should approximate(Kilowatts(expectedMinPower))
               maxPower should approximate(Kilowatts(expectedMaxPower))
           }
+      }
+    }
+
+    "Handle operating correctly" in {
+      val ambientTemperature = Celsius(10)
+
+      val cases = Table(
+        (
+          "tick",
+          "requiredDemandHouse",
+          "expectedHpQDot",
+          "expectedTick",
+        ),
+        //  (0, 0d, 0d, Some(4000)),
```

Comment:
Some more cases here?

Reply:
Oh, I missed this one! 

But I see just the two cases, do you have any other case in mind?